include/vkgcDefs.h

```diff
@@ -748,6 +748,7 @@ struct GraphicsPipelineBuildInfo {
   NggState nggState;       ///< NGG state used for tuning and debugging
   PipelineOptions options; ///< Per pipeline tuning/debugging options
   bool unlinked;           ///< True to build an "unlinked" half-pipeline ELF
+  bool dynamicVertexStride; ///< Dynamic Vertex input Stride is enabled.
```

Comment:
Please bump the minor version and leave a comment to describe what your change of VKGC interface is (See the beginning of this file).